webapp/loggedin/surveyViews/surveyForm/navigation/formEntryActions.js

```diff
@@ -27,32 +27,34 @@ const RecordEntryButtons = (props) => {
 
   const { i18n } = useContext(AppContext)
 
+  const getStepLabel = step => console.log(step) || i18n.t(`surveyForm.step.${RecordStep.getName(step)}`)
```

Comment:
let's not leave console.log in the master branch

Reply:
right